build.sbt

```diff
@@ -70,13 +41,39 @@ lazy val root = (project in file("."))
         case e: Exception => "unknown"
       }))
     ),
-    buildInfoPackage := "app"
+
+    buildInfoPackage := "app",
+    serverLoading in Debian := Systemd,
+
+    debianPackageDependencies := Seq("openjdk-8-jre-headless"),
+    maintainer := "Digital CMS <digitalcms.dev@guardian.co.uk>",
+    packageSummary := "media-atom-maker",
+    packageDescription := """making media atoms"""
   )
 
-import com.typesafe.sbt.packager.archetypes.ServerLoader.Systemd
-serverLoading in Debian := Systemd
+lazy val uploader = (project in file("uploader"))
+  .dependsOn(common)
+  .enablePlugins(JavaAppPackaging)
+  .settings(commonSettings,
+    name := "media-atom-uploader",
+    libraryDependencies ++= Dependencies.uploaderDependencies,
+
+    topLevelDirectory in Universal := None,
+    packageName in Universal := normalizedName.value
+  )
 
-debianPackageDependencies := Seq("openjdk-8-jre-headless")
-maintainer := "Digital CMS <digitalcms.dev@guardian.co.uk>"
-packageSummary := "media-atom-maker"
-packageDescription := """making media atoms"""
+lazy val root = (project in file("root"))
+  .aggregate(common, app, uploader)
+  .enablePlugins(RiffRaffArtifact)
+  .settings(
+    riffRaffBuildIdentifier := Option(System.getenv("CIRCLE_BUILD_NUM")).getOrElse("dev"),
+    riffRaffManifestBranch := Option(System.getenv("CIRCLE_BRANCH")).getOrElse("dev"),
+    riffRaffUploadArtifactBucket := Option("riffraff-artifact"),
+    riffRaffUploadManifestBucket := Option("riffraff-builds"),
+    riffRaffManifestProjectName := "media-service:media-atom-maker",
+    riffRaffArtifactResources := Seq(
+      (packageBin in Universal in app).value -> s"${(name in app).value}/${(packageBin in Universal in app).value.getName}",
```

Comment:
The `app` is a deb file so think it should be 
```
(packageBin in Debian in app).value -> s"${(name in app).value}/${(packageBin in Debian in app).value.getName}"
```

Though it might be easier to  create variables for `(packageBin in Universal in app)` and `(packageBin in Universal in uploader)` and then use them the couple times they are referred to?